test/integration/model/findAll.test.js

```diff
@@ -501,7 +501,7 @@ describe(Support.getTestDialectTeaser('Model'), () => {
         expect(users[1].intVal).to.equal(10);
       });
 
-      if (['postgres', 'sqlite'].includes(dialect)) {
+      if (['postgres', 'sqlite'].includes(dialect)) { // Yugabyte not support CITEXT yet
```

Comment:
Let's add `AbstractDialect.supports.CITEXT` instead to determine whether the type is supported

Reply:
@ephys I have addressed all the other comments and only these ones which are related to adding the flag in `Abstract Dialect` are left, I wanted to know whether should I go with that as it will require to add value for that particular support in each dialect as well?

Reply:
We'll make a PR to refactor these in the next few days so you don't have to do that in this PR. We will link the PR here after it is opened